public/index.html

```diff
@@ -80,6 +83,6 @@
     </noscript>
     <div id="root"></div>
 
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js"></script>
+    <script async src="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js"></script>
```

Comment:
Seems like we load this script twice
https://github.com/onitsoft/nexchange-open-client-react/pull/694/files#r388313703

Reply:
@IntegralTeam

Reply:
First one is JS, another one is CSS.